### PR TITLE
Add flag to updatedb command for site-update script

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes/no
| Did you apply meaningful labels to the pull request? | yes/no
| Did you perform a self review first? | yes/no
| Documentation reflects changes? | yes/no
| `CHANGELOG` reflects changes? | yes/no
| Unit/Functional tests reflect changes? | yes/no
| Did you perform browser testing? | yes/no
| Risk level | Low, Medium, High
| Relevant links | JIRA issue, bug reports, security alerts, etc.

### CODE REVIEW

**Comment by ben-hamelin** [comment]
@jordan-caldwell This also adds a fix for the site install issue tracked here:
https://support.acquia.com/hc/en-us/articles/360047676154-Site-Factory-installations-fail-when-memcache-is-included
https://insight.acquia.com/support/tickets/1134572

File: factory-hooks/post-settings-php/acsfd8+.memcache.settings.php [hunk 1]
```diff
@@ -22,7 +22,9 @@
 if (getenv('AH_SITE_ENVIRONMENT') &&
   array_key_exists('memcache', $settings) &&
   array_key_exists('servers', $settings['memcache']) &&
-  !empty($settings['memcache']['servers'])
+  !empty($settings['memcache']['servers']) &&
+  !(PHP_SAPI === 'cli' && isset($_SERVER['argv']) &&
+  in_array('site-install',$_SERVER['argv']))
```
Comment (jordan-caldwell) [inline factory-hooks/post-settings-php/acsfd8+.memcache.settings.php:27]: Needs a space after comma

**Review by jordan-caldwell — CHANGES_REQUESTED** [review]
Interesting re: the flag. This looks good just one nitpick

File: factory-hooks/post-settings-php/acsfd8+.memcache.settings.php (hunk 1 above)
Comment (ben-hamelin) [reply factory-hooks/post-settings-php/acsfd8+.memcache.settings.php]: got it!

**Review by jordan-caldwell — APPROVED** [review]
Looks good!

**Pull request merged**